WhaleSpotting/Models/Request/WhaleSightingSearchRequest.cs

```diff
@@ -0,0 +1,13 @@
+namespace WhaleSpotting.Models.Request;
+
+public class WhaleSightingSearchRequest
+{
+    public string? WhaleSpecies { get; set; }
+    public string? Colour { get; set; }
+	public TailType? TailType { get; set; }
+    public WhaleSize? Size { get; set; }
```

Comment:
It may be easier to make these strings and then use logic in the backend to use them